Spelling (#192)
botpress/docs: api-reference/runtime-api/concepts.mdx

}
```

> When an integration is deployed, it can defined any message type, but can't override any of the builtin message types with a breaking schema. This means that if an integration defines a message of type `text`, the payload must be _compatible_ with `{ text: string; }`. In other words, it can add type constraints like new properties to the payload, but it cannot remove them.

Reviewer: ```suggestion
When an integration is deployed, it can define any message type, but can't override any of the built-in message types with a breaking schema. This means that if an integration defines a message of type `text`, the payload must be compatible with `{ text: string; }`. In other words, it can add type constraints (like new properties) to the payload, but it cannot remove them.
```

## User

> A user in the Runtime API is a speaker (person or entity) that can send messages in a conversation. Just like conversations, users belong to a specific integration. interacting with your bot on WhatsApp and Slack will create two different users, even if they both represent the same person with the same email.

Reviewer: ```suggestion
A user in the Runtime API is a speaker (person or entity) that can send messages in a conversation. Just like conversations, users belong to a specific integration. Interacting with your bot on WhatsApp and Slack will create two different users, even if they both represent the same person with the same email.
```

### Direction

> Unlike messages, events have no direction. They are always considered **incoming**. This is because they are always sent to the Bot, by either an integration or the bot itself. You may send an event to your bot by an external calls to the Runtime API, but doing so, still requires you to authenticate as either a bot or an integration.

Reviewer: ```suggestion
Unlike messages, events have no direction — they are always considered **incoming**. This is because they are always sent to the Bot by either an integration or the bot itself. You may send an event to your bot by an external call to the Runtime API, but doing so still requires you to authenticate as either a bot or an integration.
```
